### PR TITLE
Chart for running k3s in k8s

### PR DESCRIPTION
**Pull request by drpebcak**

### CODE REVIEW

File: charts/k3s/0.0.1/templates/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Installed!
```
Comment (cloudnautique) [inline charts/k3s/0.0.1/templates/NOTES.txt]: Kind of lame

File: charts/k3s/0.0.1/values.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    # If defined, storageClassName: <storageClass>
+    # If set to "-", storageClassName: "", which disables dynamic provisioning
+    # If undefined (the default) or set to null, no storageClassName spec is
+    # set, choosing the default provisioner.
+    storageClass:
+
+agent:
+  replicaCount: 2
+  ingressHostname: rancher.drpebcak.me
```
Comment (cloudnautique) [inline charts/k3s/0.0.1/values.yaml]: can we tie it to an internal domain.

**Comment by cloudnautique** [comment]
Do we need to watch the plugin versions? Is there something that is automated to do this?

**Comment by drpebcak** [comment]
Yeah we need to update this whenever there are plugin updates. Jenkins will show a message about important available updates for jenkins/plugins, but there is no built-in automation around doing the updates (which is probably for the best, since updating plugins can be dicey).

**Pull request merged**